### PR TITLE
refactor(console): hide manage language button in sign-in-experience guide

### PR DESCRIPTION
**Pull request by xiaoyijun**
## Summary

Hide the manage language button in the sign-in-experience guide according to the [design](https://www.figma.com/file/UlI7FAyFMuGqxS8j1vFy0D/%F0%9F%8C%8D-%5BAC%5D-SIE-Localization?node-id=1168%3A37797).

## Testing

image

### CODE REVIEW

**Review by gao-sun — COMMENTED** [review]
should be `refactor`, both PR title and commit message

File: packages/console/src/pages/SignInExperience/components/LanguagesForm.tsx [hunk 1]
```diff
@@ -13,7 +13,11 @@ import { SignInExperienceForm } from '../types';
 import ManageLanguageModal from './ManageLanguageModal';
 import * as styles from './index.module.scss';
 
-const LanguagesForm = () => {
+type Props = {
+  isInSignInExperienceGuide?: boolean;
```
Comment (gao-sun) [inline packages/console/src/pages/SignInExperience/components/LanguagesForm.tsx:17]: showManageLanguage?

Comment (xiaoyijun) [reply packages/console/src/pages/SignInExperience/components/LanguagesForm.tsx]: updated.

File: packages/console/src/pages/SignInExperience/components/GuideModal.tsx [hunk 2]
```diff
@@ -121,7 +121,7 @@ const GuideModal = ({ isOpen, onClose }: Props) => {
                     <TermsForm />
                   </div>
                   <div className={styles.card}>
-                    <LanguagesForm />
+                    <LanguagesForm isShowManageLanguage={false} />
```
Comment (charIeszhao) [inline packages/console/src/pages/SignInExperience/components/GuideModal.tsx:124]: When do we show it?

Comment (xiaoyijun) [reply packages/console/src/pages/SignInExperience/components/GuideModal.tsx:124]: In the "others" tab on the SIE page.

File: packages/console/src/pages/SignInExperience/components/LanguagesForm.tsx [hunk 3]
```diff
@@ -13,7 +13,11 @@ import { SignInExperienceForm } from '../types';
 import ManageLanguageModal from './ManageLanguageModal';
 import * as styles from './index.module.scss';
 
-const LanguagesForm = () => {
+type Props = {
+  isManageLanguageVisible?: boolean;
+};
+
+const LanguagesForm = ({ isManageLanguageVisible = true }: Props) => {
```
Comment (charIeszhao) [inline packages/console/src/pages/SignInExperience/components/LanguagesForm.tsx:20]: nit: We usually set this type of flag to `false` by default. Then you only need to add `isLanguageManagerVisible` in the "others" tab to show it

**Review by charIeszhao — APPROVED** [review]

File: packages/console/src/pages/SignInExperience/components/LanguagesForm.tsx (hunk 3 above)
Comment (xiaoyijun) [reply packages/console/src/pages/SignInExperience/components/LanguagesForm.tsx]: updated.

**Pull request merged**